t/porting/diag.t

```diff
@@ -64,7 +64,7 @@ my $source_msg_re =
    "(?<routine>\\bDIE\\b|$function_re)";
 my $text_re = '"(?<text>(?:\\\\"|[^"]|"\s*[A-Z_]+\s*")*)"';
 my $source_msg_call_re = qr/$source_msg_re(?:_nocontext)? \s*
-    \( (?: \s* Perl_form \( )? (?:aTHX_)? \s*
+    \( (?: \s* form \( )? (?:aTHX_)? \s*
```

Comment:
```suggestion
    \( (?: \s* (?: Perl_ )? form \( )? (?:aTHX_)? \s*
```
We don't want to miss out on future diagnostics just because the author used `Perl_form` instead of `form`, right?

Reply:
Yeah that's a good suggestion.